### ISSUE TITLE
Cannot read property 'parent' of null

### ISSUE THREAD

**Issue by henningstorck** [TryGhost/Ghost#10679]
## Issue Summary

I have a photo blog with lots of posts without content but only a post image. Opening the RSS feed will throw an error 500 when having posts without content.

```
ghost_1  | ERROR [2019-04-13 09:55:57] "GET /rss/" 500 301ms
ghost_1  |
ghost_1  | Cannot read property 'parent' of null
ghost_1  |
ghost_1  | Error ID:
ghost_1  |     55c267b0-5dd2-11e9-a3c9-3992109e7879
ghost_1  |
ghost_1  | ----------------------------------------
ghost_1  |
ghost_1  | InternalServerError: Cannot read property 'parent' of null
ghost_1  |     at new GhostError (/var/lib/ghost/versions/2.19.4/core/server/lib/common/errors.js:10:26)
ghost_1  |     at _private.prepareError (/var/lib/ghost/versions/2.19.4/core/server/web/shared/middlewares/error-handler.js:43:19)
ghost_1  |     at Layer.handle_error (/var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/layer.js:71:5)
ghost_1  |     at trim_prefix (/var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/index.js:315:13)
ghost_1  |     at /var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/index.js:284:7
ghost_1  |     at Function.process_params (/var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/index.js:335:12)
ghost_1  |     at next (/var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/index.js:275:10)
ghost_1  |     at Layer.handle_error (/var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/layer.js:67:12)
ghost_1  |     at trim_prefix (/var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/index.js:315:13)
ghost_1  |     at /var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/index.js:284:7
ghost_1  |     at Function.process_params (/var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/index.js:335:12)
ghost_1  |     at Immediate.next (/var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/index.js:275:10)
ghost_1  |     at Immediate._onImmediate (/var/lib/ghost/versions/2.19.4/node_modules/express/lib/router/index.js:635:15)
ghost_1  |     at runCallback (timers.js:706:11)
ghost_1  |     at tryOnImmediate (timers.js:676:5)
ghost_1  |     at processImmediate (timers.js:658:5)
ghost_1  |
ghost_1  | TypeError: Cannot read property 'parent' of null
ghost_1  |     at Function.exports.update (/var/lib/ghost/versions/2.19.4/node_modules/cheerio/lib/parse.js:55:26)
ghost_1  |     at module.exports (/var/lib/ghost/versions/2.19.4/node_modules/cheerio/lib/parse.js:17:11)
ghost_1  |     at Function.exports.load (/var/lib/ghost/versions/2.19.4/node_modules/cheerio/lib/static.js:22:14)
ghost_1  |     at Object.makeAbsoluteUrls (/var/lib/ghost/versions/2.19.4/core/server/services/url/utils.js:396:33)
ghost_1  |     at generateItem (/var/lib/ghost/versions/2.19.4/core/server/services/rss/generate-feed.js:25:40)
ghost_1  |     at feedPromise.then (/var/lib/ghost/versions/2.19.4/core/server/services/rss/generate-feed.js:92:26)
ghost_1  |     at tryCatcher (/var/lib/ghost/versions/2.19.4/node_modules/bluebird/js/release/util.js:16:23)
ghost_1  |     at Promise._settlePromiseFromHandler (/var/lib/ghost/versions/2.19.4/node_modules/bluebird/js/release/promise.js:512:31)
ghost_1  |     at Promise._settlePromise (/var/lib/ghost/versions/2.19.4/node_modules/bluebird/js/release/promise.js:569:18)
ghost_1  |     at Promise._settlePromiseCtx (/var/lib/ghost/versions/2.19.4/node_modules/bluebird/js/release/promise.js:606:10)
ghost_1  |     at _drainQueueStep (/var/lib/ghost/versions/2.19.4/node_modules/bluebird/js/release/async.js:142:12)
ghost_1  |     at _drainQueue (/var/lib/ghost/versions/2.19.4/node_modules/bluebird/js/release/async.js:131:9)
ghost_1  |     at Async._drainQueues (/var/lib/ghost/versions/2.19.4/node_modules/bluebird/js/release/async.js:147:5)
ghost_1  |     at Immediate.Async.drainQueues [as _onImmediate] (/var/lib/ghost/versions/2.19.4/node_modules/bluebird/js/release/async.js:17:14)
ghost_1  |     at runCallback (timers.js:705:18)
ghost_1  |     at tryOnImmediate (timers.js:676:5)
```

## To Reproduce

1. Create a blog post without content but a post image.
2. Open the RSS feed.
3. The error shown above is thrown.

## Technical details

* Ghost Version: 2.19.4
* Node Version: 10.15.3
* Browser / OS: Google Chrome 73.0.3683.103 / Windows 10 
* Database: MySQL 5.7.25

**Comment by rishabhgrg** [comment]
@henningstorck This is fixed in latest release(2.20.1), thanks again for the report. 😄 🎉

**Comment by couchsoft** [comment]
I see that these bug has been fixed in 2.20.1 but I now found it happen again in 2.37.0. I have this empty post:
![grafik](https://user-images.githubusercontent.com/22036400/70177716-2dc8ba80-16db-11ea-94e1-37dc2324b2b2.png)

It results in the following error:

```
[2019-12-04 20:13:42] ERROR "GET /rss/" 500 953ms
 Cannot read property 'parent' of null
 Error ID:
     911d6f30-16d2-11ea-8ac9-539afa00b3bf

----------------------------------------
 InternalServerError: Cannot read property 'parent' of null
     at new GhostError (/var/lib/ghost/versions/2.37.0/core/server/lib/common/errors.js:10:26)
     at _private.prepareError (/var/lib/ghost/versions/2.37.0/core/server/web/shared/middlewares/error-handler.js:51:19)
     at Layer.handle_error (/var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/layer.js:71:5)
     at trim_prefix (/var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/index.js:315:13)
     at /var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/index.js:284:7
     at Function.process_params (/var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/index.js:335:12)
     at next (/var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/index.js:275:10)
     at Layer.handle_error (/var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/layer.js:67:12)
     at trim_prefix (/var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/index.js:315:13)
     at /var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/index.js:284:7
     at Function.process_params (/var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/index.js:335:12)
     at Immediate.next (/var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/index.js:275:10)
     at Immediate._onImmediate (/var/lib/ghost/versions/2.37.0/node_modules/express/lib/router/index.js:635:15)
     at runCallback (timers.js:706:11)
     at tryOnImmediate (timers.js:676:5)
     at processImmediate (timers.js:658:5)
     at process.topLevelDomainCallback (domain.js:126:23)
 TypeError: Cannot read property 'parent' of null
[..]
```

When I delete the post or set it back to draft state the rss feed is rendered perfectly fine...
